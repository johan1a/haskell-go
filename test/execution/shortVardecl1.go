package main


func main() {
    x := 5
    fmt.Println(x)
    y := "yo"
    fmt.Println(y)
    b := true
    fmt.Println(b)
    bx := false
    fmt.Println(bx)
}
