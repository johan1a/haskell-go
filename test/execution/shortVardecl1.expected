5
"yo"
true
false
